conf/httpd.conf.d/httpd.admin

```diff
@@ -103,6 +103,8 @@ if (defined($management_network->{'Tip'}) && $management_network->{'Tip'} ne '')
 $User = "pf";
 $Group = "pf";
 
+$PerlChildInitHandler = "pf::WebAPI::ChildInitHandler";
```

Comment:
Add the following to all the httpd conf files
PerlChildInitHandler  pf::WebAPI::InitHandler::child_init